Builds/VisualStudio/stellar-core.vcxproj

```diff
@@ -391,6 +391,7 @@ exit /b 0
     <ClCompile Include="..\..\src\history\FileTransferInfo.cpp" />
     <ClCompile Include="..\..\src\history\HistoryArchive.cpp" />
     <ClCompile Include="..\..\src\history\HistoryArchiveManager.cpp" />
+    <ClCompile Include="..\..\src\history\HistoryArchiveReportWork.cpp" />
```

Comment:
forgot to update the other file ".filters"

Reply:
updated